### PR TITLE
create user-settings

### PR DESCRIPTION
**Pull request by jsbd29**
I think I resolved the conflicts

### CODE REVIEW

File: ClientApp/src/App.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
-import React, { Component } from 'react';
-import { Route, Routes } from 'react-router-dom';
-import AppRoutes from './AppRoutes';
-import { Layout } from './components/Layout';
-import './index.css';
+import React, { Component } from "react";
```
Comment (Shan0x) [inline ClientApp/src/App.js:1]: I would prefer if we kept using single quotes rather than double quotes.

File: ClientApp/src/components/Home.js [hunk 2]
```diff
@@ -1,11 +1,11 @@
+
+=======
```
Comment (Shan0x) [inline ClientApp/src/components/Home.js:2]: what is this for line for? It throws an unexpected token error.

File: ClientApp/src/AppRoutes.js [hunk 3]
```diff
@@ -5,11 +5,12 @@ import { Fundraisers } from "./components/fundraiser/Fundraisers";
 import { Donate } from "./components/Donate";
 import { Dashboard } from "./components/user/Dashboard";
 import { CreateFundraisers } from "./components/fundraiser/CreateFundraiser";
+import { Settings } from "./components/Settings";
```
Comment (Shan0x) [inline ClientApp/src/AppRoutes.js:8]: This module can't be found in ./components/Settings. You have Settings.jsx in ../src

File: ClientApp/src/components/NavMenu.js [hunk 4]
```diff
@@ -26,17 +33,29 @@ export class NavMenu extends Component {
...
               </NavItem>
               <NavItem>
+                <NavLink tag={Link} className="text-dark" to="/settings">
+                  Settings
+                </NavLink>
                 <NavLink tag={Link} className="text-dark" to="/sign-up">Create a Fund</NavLink>
               </NavItem>
               <NavItem>
-                <NavLink tag={Link} className="text-dark" to="/login">Login</NavLink>
+                <NavLink tag={Link} className="text-dark" to="/login">Login</NavLink
```
Comment (Shan0x) [inline ClientApp/src/components/NavMenu.js:58]: Parsing error. Missing a >.

**Review by crjpb3 — CHANGES_REQUESTED** [review]
NavMenu.js Line 58: Missing an enclosing ">" symbol on "</NavLink"

Home.js Line 2: sequence of "=" symbols; doesn't seem to be code, should be commented?

Error in AppRoutes.js while trying to locate ./components/Settings. Settings seems to be located in the src folder rather than the components folder

**Review by Shan0x — CHANGES_REQUESTED** [review]
There's a few things that need to be changed. I am unable to view the settings pages or find a way to view it to see if it's viewable - this is most likely because you pointed your Settings page to the wrong folder. I've written some comments on some lines that I didn't quite understand. Please feel free to comment back or message me on discord regarding your changes or if you have any questions. Thank you. :)

File: ClientApp/src/components/Home.js (hunk 2 above)
Comment (jsbd29) [reply ClientApp/src/components/Home.js:2]: I think is some mess I made  from github   :(

File: ClientApp/src/components/NavMenu.js (hunk 4 above)
Comment (jsbd29) [reply ClientApp/src/components/NavMenu.js:58]: sorry !

**Review by jsbd29 — COMMENTED** [review]
I am sorry for all the mistakes guys!

**Pull request closed without merging**